packages/mantine/src/components/prompt/PromptConfirmButton.tsx

```diff
@@ -1,5 +1,6 @@
 import {Button, CompoundStylesApiProps, factory, Factory, useProps} from '@mantine/core';
 import {ButtonProps} from '../button/Button';
+import {ButtonWithDisabledTooltip} from '../button/ButtonWithDisabledTooltip';
```

Comment:
Does it work if you simply import button from ../button/Button instead of importing it from Mantine?
```suggestion
import {CompoundStylesApiProps, factory, Factory, useProps} from '@mantine/core';
import {Button, ButtonProps} from '../button/Button';
```

Reply:
🤦 probably